app/views/items/show.html.erb

```diff
@@ -0,0 +1,10 @@
+
+<%= @item.name  %>
+<%= @item.description %>
+<%= @item.price %>
+<%= @item.available %>
+<%= @item.category ? "available" : "unavailable" %>
```

Comment:
This should be the category text. Available takes the thing you are doing here :)